### ISSUE TITLE
Improved region-comment folding support

### ISSUE THREAD

**Issue by Bernd-L** [fwcd/vscode-kotlin#76]
### Motivation
Having the same code folding syntax supported in Android Studio and VS Code would be great.  
It's just really annoying that they're not the same.

### Description
Code like

```kotlin
//region Some name here
foo()
//endregion
```

is understood by Android Studio, and VS Code (if set to or Rust using the RLS extension). This means it's possible for
VS Code to understand this syntax, if the language extension supports it.

However, when VS Code uses this extension to handle Kotlin, the code can no longer be folded.

### Proposed change

Accept both

```kotlin
// #region regionName
vsCodeStyle()
// #endregion
```

and

```kotlin
//region regionName
androidStudioStyle()
//endregion
```

syntax comments.

If I have time, I might open a PR myself, let me know if that'd help.

# TL;DR

`// #region regionName` & `// #endregion` works, but `//region regionName` & `//endregion` doesn't;  
please fix/improve

**Comment by Bernd-L** [comment]
I just found <https://github.com/fwcd/kotlin-language-server/blob/04da62f9dcbb937cb981a9d2cc14bf346a15eadf/grammars/kotlin.configuration.json> (other repository), which appears to be the file which needs to be modified.

I intend to leave this issue open, until the fix is available in this extension; let me know if that's a problem

**Comment by Bernd-L** [comment]
https://github.com/fwcd/kotlin-language-server/pull/307 is now merged :rocket: :tada:
